domains/neovoid.is-cool.dev.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Project Description",
+
+  "domain": "is-cool.dev",
+  "subdomain": "neovoid",
+
+  "owner": {
+    "email": "neovoid@envs.net "
+  },
+
+  "record": {
+    "CNAME": ["exozy.me"]
```

Comment:
Invalid CNAME format, CNAME should be a string not an array.

Reply:
like this without square bracket?

Reply:
Yep, latest commit looks good.